ui/src/components/admin/leadershipTeam/AdminLeadershipTeamContainer.js

```diff
@@ -1,125 +1,368 @@
+          RESET
+        </Button>
+        <Button
+          leftIcon={id !== '' ? <EditIcon /> : <AddIcon />}
+          colorScheme="teal"
+          onClick={onOpen}
+        >
+          {id !== '' ? 'EDIT' : 'ADD'}
+        </Button>
+      </Flex>
+
+      <Stack direction={['column']} w="100%">
+        <Box w={['100%']}>
+          <LeadershipTeamGrid teams={teams} setSelected={setSelected} />
+        </Box>
+
+        <Drawer
+          isOpen={isOpen}
+          placement="right"
+          onClose={onClose}
+          initialFocusRef={firstField}
+          size="lg"
+        >
+          <DrawerOverlay />
+          <DrawerContent>
+            <DrawerCloseButton />
+
+            <DrawerHeader></DrawerHeader>
+
+            <DrawerBody>
+              <form id="lt-form" onSubmit={onSubmit}>
+                <FormControl>
+                  <FormLabel>Season From</FormLabel>
+                  <Input
+                    ref={firstField}
+                    type="date"
+                    value={seasonFrom}
+                    onChange={(e) => setSeasonFrom(e.target.value)}
+                  />
+                </FormControl>
+
+                <FormControl>
+                  <FormLabel>Season To</FormLabel>
+                  <Input
+                    type="date"
+                    value={seasonTo}
+                    onChange={(e) => setSeasonTo(e.target.value)}
+                  />
+                </FormControl>
+
+                <FormControl>
+                  <FormLabel>Campus</FormLabel>
+                  <Select
+                    size="sm"
+                    borderRadius="5"
+                    value={campus}
+                    placeholder="Please fill in this field"
+                    onChange={(e) => setCampus(e.target.value)}
+                  >
+                    {campusList.map((item) => {
+                      return <option key={item}>{item}</option>;
+                    })}
+                  </Select>
+                </FormControl>
+
+                <FormControl>
+                  <FormLabel>Lifestage</FormLabel>
+                  <Select
+                    size="sm"
+                    borderRadius="5"
+                    value={lifestage}
+                    placeholder="Please fill in this field"
+                    onChange={(e) => setLifestage(e.target.value)}
+                  >
+                    {lifestageList.map((item) => {
+                      return <option key={item}>{item}</option>;
+                    })}
+                  </Select>
+                </FormControl>
+
+                <FormControl>
+                  <FormLabel>LIFE Group</FormLabel>
+                  <Input
+                    type="text"
+                    value={lifeGroup}
+                    onChange={(e) => setLifeGroup(e.target.value)}
+                  />
+                </FormControl>
+
+                <FormControl>
+                  <FormLabel>Leaders</FormLabel>
```

Comment:
should this be labeled as leaders' emails and then should we check that these are all emails?

Reply:
since the actual field seems to be called leader's emails in the file below